### PR TITLE
Ui design update

### PR DESCRIPTION
**Pull request by Jeffchiucp**
Ui design update for each pages

### CODE REVIEW

File: public/javascripts/js-map.js [hunk 1]
```diff
@@ -76,9 +77,6 @@ function getLocation() {
         }, function() {
             handleLocationError(true, infoWindow, map.getCenter());
         });
-    } else {
```
Comment (anselb) [inline public/javascripts/js-map.js]: What is the reason for deleting these 3 lines?

File: views/itineraries-show.jade [hunk 2]
```diff
@@ -1,15 +1,15 @@
 extends layout
 
 block content
-
+    if user
```
Comment (anselb) [inline views/itineraries-show.jade]: I don't think this line of code does what it is suppose to do. The value of 'user' must be passed in from the server side, and I don't believe there is a value called 'user'.

File: views/index.jade [hunk 3]
```diff
@@ -7,7 +7,9 @@ block content
         // DISPLAY LOGIN OR REGISTER WHEN NOT LOGGED IN
         if (!currentUser)
             //p.itinerary-add-message Login or register
-        h1.myclaas Itinerary List
+        center
+          h1.myclaas List of Awesome Activity Planners
```
Comment (anselb) [inline views/index.jade]: I don't think the class 'myclaas' is in the stylesheet; I also couldn't find 'myclass', so I am wondering if we need to label this div with that class name. 
While I do think that the content of this header aligns with the name of our app, the actual name of our app is not correct in the top left of the nav bar.

File: slider.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+//Created by Cole Hemp(GamePlayerCole)
```
Comment (anselb) [inline slider.js]: This file doesn't seem to add anything to our project. If it does, please point out the functionality in a comment below the other comments.

File: views/activities-new.jade [hunk 5]
```diff
@@ -2,15 +2,16 @@ extends layout
...
             form(action='/itineraries/#{itineraryId}/activities/new')
+              center
                 .form-group
                     label(for='search-term')="Search Term (1 Key Word)"
                     input#search-term(placeholder=' food, drink, or theater', name='term', type='text', value='#{term}')
                 .form-group
                     label(for='search-location')="Location (City or Zip Code)"
                     input#search-location(placeholder= ' SF, NY, DC, or 94107',name='location', type='text', value='#{location}')
-                .text-right
+                    br
```
Comment (anselb) [inline views/activities-new.jade]: Line breaks should not be used like this. We can temporarily use them for spacing, but spacing should be done in the stylesheet.

File: views/itineraries-show.jade (hunk 2 above)
Comment (Jeffchiucp) [reply views/itineraries-show.jade]: you are right, I needed to create another branch to fix the user validation check

File: views/index.jade (hunk 3 above)
Comment (Jeffchiucp) [reply views/index.jade]: Yes. I added a quick fix and created the myclaas in the stylesheet,  So I can align the center

**Pull request merged**